DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js

```diff
@@ -87,13 +88,13 @@ module.exports = function(ngModule) {
 
     this.initDatatable();
 
-    $('#example tbody').on('click', 'td.edit-table-entry', function ($event) {
+    $('#' + tableId + ' tbody').on('click', 'td.edit-table-entry', function ($event) {
```

Comment:
In this case:  ``$(`#${tableId} tbody`)``.  From here on, I'll just note strings that should be fixed.